### PR TITLE
In source now-defunct Pants team fork of Jar Jar Links

### PR DESCRIPTION
**Pull request by er1c**
Addresses #11 (at least partially) and fixing https://github.com/eed3si9n/jarjar-abrams/pull/15

* Created a copy of https://github.com/pantsbuild/jarjar with git-history as `jarjar` subproject
* Updated `build.sbt` and `README` to reference the `jarjar` project and where it came from
* Added a `KeepProcessorTest` and fix.
* Added a best-guess for `BigtableIO$Write.class` since no version was available (test passes though)
* Moved the `org.pantsbuild.jarjar.integration` into `jarjar/src/it` and removed the `pants` dependency

See updated [README.markdown](https://github.com/er1c/jarjar-abrams/blob/add-pants-jarjar/README.markdown)

### CODE REVIEW

**Comment by er1c** [comment]
Verified that this fixes the `sbt-assembly` test failure, but I'd like to do some more tests, I need to split the actual fix out from the big commit too to make easier to review.

**Comment by er1c** [comment]
@alexarchambault do you want to review this?  what was the issue with the `ASM`?

File: README.markdown [hunk 1]
```diff
@@ -3,6 +3,11 @@ Jar Jar Abrams
 
 Jar Jar Abrams is an experimental Scala extension of [Jar Jar Links][links] a utility to shade Java libraries.
 
+Pants Jar Jar
```
Comment (eed3si9n) [inline README.markdown]:
```suggestion
Jar Jar Links
```

File: README.markdown [hunk 2]
```diff
@@ -3,6 +3,11 @@ Jar Jar Abrams
...
+============
+
+[Pants Jar Jar](https://github.com/pantsbuild/jarjar) was migrated at commit 57845dc73d3e2c9b916ae4a788cfa12114fd7df1, dated Oct 28, 2018.
```
Comment (eed3si9n) [inline README.markdown]:
```suggestion
In 2004, herbyderby (Chris Nokleberg) created a tool called Jar Jar Links that can repackage Java libraries.

Now defunct [fork of Jar Jar Links by Pants](https://github.com/pantsbuild/jarjar) was in-sourced at commit 57845dc73d3e2c9b916ae4a788cfa12114fd7df1, dated Oct 28, 2018.
```

**Comment by er1c** [comment]
✅ with the CI tests now running/passing

**Review by eed3si9n — APPROVED** [review]
Thanks

**Pull request merged**